kornia/geometry/dsnt.py

```diff
@@ -51,21 +52,21 @@ def spatial_softmax_2d(
     return x_soft.view(batch_size, channels, height, width)
 
 
-def spatial_softargmax_2d(
+def spatial_expectation_2d(
```

Comment:
to follow other naming conventions (gaussian_blurd2d, conv_soft_argmax2d) I owuld remove underscore -> `spatial_expectation2d`

Reply:
Does this mean that `spatial_softmax_2d` and `render_gaussian_2d` also need to be renamed?

Reply:
yes